tests/unit/metrics/export/test_point.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from datetime import datetime
+from opencensus.metrics.export import point as point_module
+from opencensus.metrics.export import summary as summary_module
+from opencensus.metrics.export import value as value_module
+
+
+class TestPoint(unittest.TestCase):
+
+    def setUp(self):
+        self.double_value = value_module.Value.double_value(55.5)
+        self.long_value = value_module.Value.long_value(9876543210)
+        self.timestamp = datetime.utcnow().isoformat() + 'Z'
```

Comment:
Might want to freeze this to avoid flaky tests if something does go wrong parsing a particular time.